### DISCUSSION TITLE
Add support info download button back to help page

### DISCUSSION THREAD

**Discussion by MichaelDvP** [emsesp/EMS-ESP32#1489] [General]
On new issues the requested support info was very  rarly added, so it looks like a good ifdea to move the download button to the "Upload/Download" page. But since that i've never seen a support info anymore. Also asking for it is completly ignored by the users. (I will ignore issues without support info in future). 
- The Upload/Download page is mainly for config-json that can be uploaded again, the support info does not fit in this scheme.
- The download section starts with a warning not to share sensitive information, but support info should be shared.

In addition we have now the `api/system/allvalues` that could also help debugging. 
I suggest to move the download button back to help page, and show it more prominent.
![grafik](https://github.com/emsesp/EMS-ESP32/assets/59284019/026806c0-aa5f-4e7a-ae43-0f42f1d298b8)
Or (and?)  make it better recognized on the download page.
![grafik](https://github.com/emsesp/EMS-ESP32/assets/59284019/2c4b807d-173b-47df-9592-390bfe454231)

**Comment by proddy** [comment]
it does not harm to add it back - I'll move the code into a separate react component so it can be shared in the Upload/Download page.

However, I don't think it'll make a big difference. Humans are generally lazy.

What I'm thinking is doing the same as Tasmota and Home Assistant's GitHub repos which both have stricter checks. If there is no system information attached to the issue it gets automatically rejected (https://github.com/arendst/Tasmota/blob/development/.github/issue-close-app.yml) using https://github.com/offu/close-issue-app

**Comment by proddy** [comment]
added a PR with your suggestions. the GH bot doesn't check for attachments, just the format of the template, so it's less interesting.
